### PR TITLE
Add contribution section and Jekyll setup help

### PR DESCRIPTION
**Pull request by sven1103**
Fixes:
- A minor thing in the index.md, because it is not getting processed by Jekyll without the layout header.
- My name :D

New:
- A introduction how to use Jekyll to build and render the website locally for development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,48 @@ If you prefer, you can read all of the website content on GitHub.com instead:
 * [About](about.md)
 
 Otherwise, head over to https://nf-core.github.io and enjoy!
+
+## Contributing to nf-core.github.io
```
Comment (ewels) [inline README.md:20]: Maybe easier to just link to the GitHub documentation? https://help.github.com/articles/setting-up-your-github-pages-site-locally-with-jekyll/

File: index.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+---
+layout: default
```
Comment (ewels) [inline index.md:2]: I have a feeling that this is only required with the vanilla Jekyll engine? I haven’t needed it running with the GitHub pages instructions above. But probably doesn’t hurt..

Comment (sven1103) [reply index.md:2]: I will test it :)

Comment (sven1103) [reply index.md:2]: I am a Jekyll n00b

File: README.md (hunk 1 above)
Comment (sven1103) [reply README.md:20]: well, it worked. I followed the Jekyll documentation on https://jekyllrb.com/ and failed at first.

File: index.md (hunk 2 above)
Comment (sven1103) [reply index.md:2]: I left it there, as it works with Jekyll vanilla and the GitHub setup.

**Comment by ewels** [comment]
Thanks @sven1103! 👍

**Pull request merged**